lib/charms/nginx_ingress_integrator/v0/ingress.py

```diff
@@ -161,7 +157,20 @@ def _config_dict_errors(self, update_only=False):
             self.model.unit.status = BlockedStatus(blocked_message)
             return True
         if not update_only:
-            missing = [x for x in REQUIRED_INGRESS_RELATION_FIELDS if x not in self.config_dict]
+            missing = [
+                config_key
+                for config_key in REQUIRED_INGRESS_RELATION_FIELDS
+                if config_key not in self.config_dict
+            ]
+            # Mapp configuration keys to new keys
+            missing = list(
+                map(
+                    lambda config_key: RELATION_INTERFACES_MAPPINGS[config_key]
+                    if config_key in RELATION_INTERFACES_MAPPINGS
+                    else config_key,
+                    missing,
+                )
+            )
```

Comment:
I think this is going to mean that we're returning the new key names as what's missing, but the charm library as currently written still requires the old values, so that's going to be a bit confusing.

What's changing is what's being passed over the relation, not what users are asked to use when working with this charm library.

Reply:
Thanks, fixed